applications/luci-app-firewall/htdocs/luci-static/resources/tools/firewall.js

```diff
@@ -211,6 +211,15 @@ return L.Class.extend({
 		return rv;
 	},
 
+	fmt_family: function(family) {
+		if (family == 'ipv4')
+			return _('IPv4');
+		else if (family == 'ipv6')
+			return _('IPv6');
+		else
+			return _('IPv4/6');
```

Comment:
Can you change this to `IPv4 and IPv6` as this was the old translation label in the Lua page.

Reply:
Changed